run.sh

```diff
@@ -0,0 +1 @@
+npm run tattoo -- $@
```

Comment:
do you want to put this in package.json/scripts so that it has a hope of working on windows?